source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +478,49 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of entity names to an array of
+    event names.
+    
+    For each entity name, the test runner MUST create the respective entity
+    with a type of "event list", as described in `Supported Entity Types`_.
+    If the entity already exists (such as from a previous
+    ``storeEventsAsEntities`` declaration from another client),
+    the test runner MUST raise an error.
+    
+    The test runner MUST set up an event subscriber for each event named.
+    The event subscriber MUST serialize the events it receives into a document,
+    using the documented properties of the event as field names, and append
+    the document to the list stored in the specified entity.
+    Additionally, the following fields MUST be stored with each
+    event document:
+    
+    - ``name``: The name of the event, such as ``PoolCreatedEvent``.
+      The name of the event MUST be the name used in the respective
+      specification that defines the event in question.
+    - ``observedAt``: The floating-point time since the Unix epoch
+      when the event was observed by the test runner.
+        
+    Currently, only the following
+    `CMAP <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`__
+    events MUST be supported:
```

Comment:
@p-mongo: Am I misremembering, or did you recently say that command monitoring events _would_ need to be re-introduced to `storeEventsAsEntities`? If so, does this section need an update?

Reply:
I added command events back. Since command events are defined more flexibly, I also added a note requesting the drivers use the field names used in the specification even if their event objects have different fields, as well as mentioning type conversion to BSON types if event objects' fields are not of BSON types.